php-practice/section5.php

```diff
@@ -158,7 +168,15 @@ function printLines(array $lines) {
  * So if $min is 5 and $max is 10, it would print out: 5 6 7 8 9 10
  */
 
+    function getValuesBetween() {
+      $max = 10;
 
+      for ($min= 5; $min <= $max; $min++) {
+        echo $min . PHP_EOL;
+      }
+}
+
+echo getValuesBetween();
```

Comment:
I figured that was how I was supposed to do it. I couldn't get that to work. I know that we want to give the function the two parameters of $min and $max but I can't get it to not return a syntax error when I try to call the function.